.circleci/config.yml

```diff
@@ -1168,6 +1168,71 @@ jobs:
                       make postman
             - notify-on-failure
 
+    publish_rpm_packages:
+        parameters:
+            dry_run:
+                default: true
+                type: boolean
+            graviteeio_version:
+                type: string
+                default: $GRAVITEEIO_VERSION
```

Comment:
Does it really work? Because this variable will be only available at runtime

Reply:
$GRAVITEEIO_VERSION removed